configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java

```diff
@@ -313,6 +322,13 @@ public boolean useFastValueTensorImplementation() {
         @Override public boolean useAccessControlTlsHandshakeClientAuth() { return useAccessControlTlsHandshakeClientAuth; }
 
         @Override public double jettyThreadpoolSizeFactor() { return jettyThreadpoolSizeFactor; }
+
+        @Override public boolean useAsyncMessageHandlingOnSchedule() { return false; }
+
+        @Override public int contentNodeBucketDBStripeBits() { return 0; }
+
+        @Override
+        public int mergeChunkSize() { return 0; }
```

Comment:
Should this return the default 4MiB value instead?

Reply:
Argh, using intellij generate metod adds a default that is easily forgotten.
Fixed and added test.